examples/policy_triggers/playbooks/delete.yaml

```diff
@@ -0,0 +1,8 @@
+---
+- hosts: all
+  gather_facts: false
+
+  tasks:
+    - name: OpenStack tasks
+      debug:
+        msg: Insert OpenStack tasks

```

Comment:
No, this indicates that OpenStack tasks should be there for real use cases, but we don't have them because testing would then require a special environment.